Source/DafnyCore/DafnyCore.csproj

```diff
@@ -23,6 +23,7 @@
   </PropertyGroup>
 
   <ItemGroup>
+      <PackageReference Include="dafny.msbuild" Version="1.0.0" />
```

Comment:
> I'm concerned that later changes/additions to the grammar might introduce missing/incorrect ownedTokens

In this PR, `ownedTokens` is under-specified, meaning not every token is assigned to a node, although I did my best so that it will be the case.